src/app/multiwallet/multiwallet.service.ts

```diff
@@ -34,28 +33,38 @@ export class MultiwalletService implements OnDestroy {
     // http request
     // subscribe to server side stream
     // and load the wallets in _list.
-    this.timer.takeWhile(() => !this.destroyed)
-      .subscribe(() => {
-        this._http.get('wallets://lists')
-        .map((response: any) => response.wallets)
-        .do((wallets) => wallets.forEach(w => w.fakename = w.name.replace('wallet_', '')))
-        .subscribe((wallets: IWallet[]) => this._list.next(wallets));
-      })
+    this.timer.takeWhile(() => !this.destroyed).subscribe(() => {
+      this.rpc.call('listwallets').subscribe(
```

Comment:
rm the electron backend code if you're going this route and remove the chokadir dependency.